src/components/NavBar/TopicDropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactElement, useState } from 'react';
+import { ChevronDownIcon } from '@heroicons/react/solid';
+import { DropdownList, ItemType } from '../Dropdown';
+import styles from './NavBar.module.css';
+
+const topics: ItemType[] = [
+  {
+    id: 'topic-general',
+    label: 'General Science',
+  },
+  {
+    id: 'topic-astrophysics',
+    label: 'Astrophysics',
+  },
+  {
+    id: 'topic-heliophysics',
+    label: 'Heliophysics',
+  },
+  {
+    id: 'topic-planetary',
+    label: 'Planetary Science',
+  },
+  {
+    id: 'topic-earth',
+    label: 'Earth Science',
+  },
+  {
+    id: 'topic-biophysical',
+    label: 'Biological & Physical Science',
+  },
+];
+
+export const TopicDropdown = (): ReactElement => {
+  const [selectedTopic, setSelectedTopic] = useState<ItemType>(topics[0]);
+
+  const getLabelNode = (label: string) => (
+    <div
+      id="topicSelector"
+      className={`${styles['navbar-bg-color']} ${styles['navbar-text-color']} flex items-center justify-between w-64 border border-gray-50 border-opacity-50 rounded-sm cursor-pointer`}
```

Comment:
We may want to use `clsx` just to clean up these classes instead of using template strings.  But not a show stopper 😃